examples/CommonAcControl/platformio.ini

```diff
@@ -15,3 +15,11 @@ board = nodemcuv2
 [env:esp32dev]
 platform = espressif32
 board = esp32dev
+
+; Build the program forcing the compiler to treat all warnings as errors.
+[env:shakedown]
```

Comment:
maybe use same name as in SmartIRRepeater/platformio.ini  to be consistent?

Reply:
Good point. Fixed.